integr8ly/cluster-service
# feat: terminate watch mode when all resources complete

aka "and now my watch has ended" ;)

# Verification Steps

* Use CRO to create some resources in AWS. Particularly a type that takes some time to delete (such as a postgres or an elasticache instance)
* run the CLI tool using `./cluster-service cleanup <cluster-id> --dry-run=false --watch`

Verify that:

* The cli tool prints a new report table every 30 seconds.
* The process ends once all the resources are reported as deleted
